src/foam/nanos/auth/UpdatePassword.js

```diff
@@ -51,7 +60,17 @@ foam.CLASS({
         class: 'foam.u2.view.PasswordView',
         passwordIcon: true
       },
-      minLength: 10
```

Comment:
Don't remove this, let's continue to figure out why it's not working or available.

Reply:
I would expect this is accessible in validation calls.